pages/_app.tsx

```diff
@@ -1,3 +1,5 @@
+import "@fontsource/roboto/400.css";
```

Comment:
Should we import all font stuff to a file and export it out from there so we standardize the font we use? I feel like people won't know what font to use and we'll have diff fonts throughout the app

Reply:
the point of this file is to standardize the font to use - the font(s) defined here will be used by default